GPU/GPUCommon.h

```diff
@@ -67,6 +67,18 @@ struct TransformedVertex {
 	}
 };
 
+inline bool IsTrianglePrim(GEPrimitiveType prim) {
+	// TODO: KEEP_PREVIOUS is mistakenly treated as TRIANGLE here... This isn't new.
```

Comment:
Well, at one point I think it was pretty consistently used only after the prim had been converted to a "real" prim type.  So while this function may have not supported the new way it's been used before and still not support the new way it's being used, I'm not quite sure the problem is not new.

That said, the only game I know of for sure using this I think only uses them for triangles, so probably not a big deal...

-[Unknown]